src/main/kotlin/no/nav/dagpenger/inntekt/InntektApi.kt

```diff
@@ -222,7 +223,8 @@ fun Application.inntektApi(
                 if (HttpStatusCode.fromValue(cause.status).isSuccess()) HttpStatusCode.InternalServerError else HttpStatusCode.fromValue(
                     cause.status
                 )
-            LOGGER.error("Request failed against inntektskomponenten", cause)
+            sikkerLogg.error(cause) { "Request failed against inntektskomponenten" }
```

Comment:
Så i går at inntektskomponenten har med ident i feilene enkelte ganger

Reply:
https://logs.adeo.no/app/kibana#/discover/doc/96e648c0-980a-11e9-830a-e17bbd64b4db/shrink-logstash-apps-prod-001439?id=ywnu6nEBO0mweIgZAaDf&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-24h,to:now))